demo/src/right-resizable-box.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React, { useState } from 'react';
+import ResizePanelHandleIcon from '@mui/icons-material/MoreVert';
+import { ResizableBox } from 'react-resizable';
+import { useWindowWidth } from '@react-hook/window-size';
+import makeStyles from '@mui/styles/makeStyles';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+
+const useStyles = makeStyles((theme) => ({
+    panel: {
+        position: 'relative',
+        boxSizing: 'border-box',
+    },
+    innerResizablePanel: {
+        flex: 'auto',
+        height: '100%',
+    },
+    resizePanelHandle: {
+        // This panel's right border looks like the panel's handle but is only a decoy.
+        // The true handle is wider than it seems, to be easier to grip, and is invisible.
+        borderRightColor: theme.palette.action.disabled,
+        borderRightStyle: 'solid',
+        borderRightWidth: theme.spacing(0.5),
+        '& .react-resizable-handle': {
+            position: 'absolute',
+            width: theme.spacing(1),
+            height: '100%',
+            top: 0,
+            right: '-' + theme.spacing(0.75),
+            cursor: 'col-resize',
+            backgroundColor: 'rgba(0, 0, 0, 0)', // The handle is invisible (alpha = 0)
+            zIndex: 5,
+        },
+    },
+    resizePanelHandleIcon: {
+        bottom: '50%',
+        right: theme.spacing(-1.75),
+        position: 'absolute',
+        color: theme.palette.text.disabled,
+        transform: 'scale(0.5, 1.5)',
+    },
+}));
+
+const RightResizableBox = (props) => {
```

Comment:
Why not if it's just for the demo, but for now it will be packaged with other dependencies in commons-ui, then in front apps which use commons-ui as dependency.

Reply:
As it is defined in the demo part, I hope its code is not exported by the `npm run build`.

Reply:
It's code no, but the dependencies are bundled with de commons-ui package I think. You moved it in devDependency, will see if it's ok.